homeassistant/components/deconz/const.py

```diff
@@ -16,6 +13,8 @@
 SUPPORTED_PLATFORMS = ['binary_sensor', 'cover',
                        'light', 'scene', 'sensor', 'switch']
 
+DECONZ_REACHABLE = 'deconz_reachable'
```

Comment:
maybe group DECONZ_* ?

Reply:
The grouping now is more in the order it is used. Maybe better to do a grouping on names. I'll think about it!